docs/dev/libvirt/README.md

```diff
@@ -286,6 +286,40 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### Console doesn't come up
+In case of libvirt there is no wildcard DNS resolution and console depends on the route which is created by auth operator.
```

Comment:
I thought I asked for this before but maybe my comment didn't make it through: Could we please refer to the issue we're working around in here and in the commit log?

Reply:
@zeenix done.